bosh_cli/spec/unit/commands/release/upload_release_spec.rb

```diff
@@ -253,6 +253,32 @@ module Bosh::Cli::Command::Release
             end
           end
 
+          context 'with --fix' do
+            before do
+              command.add_option(:fix, true)
+            end
+
+            it 'should upload the release archive along with fix option' do
+              expect(command).to receive(:upload_tarball)
+                .with(release_archive, hash_including(:fix => true))
+                .and_call_original
+              expect(director).to receive(:upload_release).with(release_archive, hash_including(:fix => true))
+              command.upload(release_archive)
+            end
+
+            it 'should ignore --name and --version' do
```

Comment:
@mariash, based on our understanding, if we provide --fix option together with --name & --version, we will ignore the check of whether the release is already uploaded, so we don't check this in the test